configuration/drugs/drugs.csv

```diff
@@ -1,35 +1,35 @@
 Uuid,Void/Retire,Name,Concept Drug,Concept Dosage Form,Strength,_version:1
-2cfadb46-140a-49a3-9ed6-cd3b64937786,0,ABC/3TC 120/60mg,ABC/3TC,Comprimé,120/60mg,
-4046a271-b3ec-4fcd-b6a8-1b17e0ea36ec,0,ABC/3TC 60/30mg,ABC/3TC,Comprimé,60/30mg,
-0456201c-adfc-4930-a527-d5efc3c408dd,0,ABC/3TC 600/300mg,ABC/3TC,Comprimé,600/300mg,
-589916a5-438e-45d6-8857-e7bb15834f7c,0,ATV/r 300/100mg,ATV/r,Comprimé,300/100mg,
-7c9507a1-f74e-438a-8471-083397e040a9,0,AZT/3TC 300/150mg,AZT/3TC,Comprimé,300/150mg,
-c0d315a9-63cc-4020-914a-9f0a7e43cfa9,0,AZT/3TC 60/30mg,AZT/3TC,Comprimé,60/30mg,
-3a06d492-d51c-4f0f-8579-4a8eb722caa4,0,AZT/3TC 600/300mg,AZT/3TC,Comprimé,600/300mg,
-7861ff41-eae5-4c1c-988c-a5dc6083888d,0,CTX 100/20mg,CTX,Comprimé,100/20mg,
+2cfadb46-140a-49a3-9ed6-cd3b64937786,0,ABC/3TC 120/60mg,ABC/3TC,Comprimé(s),120/60mg,

```

Comment:
@rubailly Why was this changed to Comprime(s)?

Reply:
You can see all the valid dosage forms running this:

select * from concept_name where concept_id in (select distinct dosage_form from drug);

Comprime(s) and all the other dosage forms in this file are not valid ones.

Reply:
```mysql> select distinct name from concept_name where concept_id in (select distinct dosage_form from drug);
+-----------------+
| name            |
+-----------------+
| Ayurvedic       |
| Inhaler         |
| Gel             |
| Injection       |
| Food Supplement |
| Cream           |
| Drops           |
| Sirop           |
| Solution        |
| Powder          |
| Ointment        |
| Lotion          |
| Jelly           |
| Comprimé        |
| Capsule         |
| Suspension      |
| Oral Rinse      |
| Mouth Paint     |
| Pessary         |
| Rotacap         |
| Nasal Solution  |
| Oral Solution   |
+-----------------+
22 rows in set (0.00 sec)
```